examples/test_util/ERft.hpp

```diff
@@ -0,0 +1,83 @@
+/*
+   Copyright 2019 Equinor ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#ifndef ERFT_HPP
+#define ERFT_HPP
+
+
+#include "EclFile.hpp"
+
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <vector>
+#include <ctime>
+#include <map>
+#include <set>
+
+
+class ERft : public EclFile
+{
+public:
+    explicit ERft(const std::string &filename);
+
+    using RftDate = std::tuple<int,int,int>;
```

Comment:
When you make `RFTDate` type it might worth it to make a struct with named fields?

Reply:
i prefer not to. i then have to write operator < and some stuff due to map usage.